tests/run_tests.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+source ../scripts/lib/shell_logger.sh
+source ../scripts/lib/sh_arg.sh
+source ./utils.sh
+
+declare LANDING_ZONES_FOLDER=""
+declare CONFIG_FOLDER=""
+declare ENVIRONMENT=""
+declare DEBUG_FLAG=false
+declare CREATE_ENV=false
+
+shArgs.arg "LANDING_ZONES_FOLDER" -z --zones PARAMETER true
+shArgs.arg "CONFIG_FOLDER" -c --config PARAMETER true
+shArgs.arg "ENVIRONMENT" -e --environment PARAMETER true
+shArgs.arg "CREATE_ENV" -ce --create-env FLAG true
+shArgs.arg "DEBUG_FLAG" -d --debug FLAG true
+
+shArgs.parse $@
+
+main() {
+  export_arm_subscription_id
```

Comment:
Minor items

* We should check for the existence of the az cli . You can use this function in scripts/lib/utils.sh
https://github.com/aztfmod/symphony/blob/master/scripts/lib/utils.sh#L8

* We should also check to ensure az is logged in, if not we can prompt the user.
You can use this function for that.
https://github.com/aztfmod/symphony/blob/master/scripts/lib/utils.sh#L16

Reply:
thanks for the suggestion , added 2 checks at the beginning of the main method in the correct order